ionoscloud/resource_dbaas_redisdb_replicaset_test.go

```diff
@@ -33,6 +35,46 @@ func TestAccDBaaSRedisDBReplicaSetBasic(t *testing.T) {
 					resource.TestCheckResourceAttr(constant.DBaaSRedisDBReplicaSetResource+"."+constant.DBaaSReplicaSetTestResource, replicaSetVersionAttribute, replicaSetVersionValue),
 				),
 			},
+			{
+				Config: redisDBReplicaSetDataSourceMatchID,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckDBaaSRedisDBReplicaSetExists(constant.DBaaSRedisDBReplicaSetResource+"."+constant.DBaaSReplicaSetTestDataSourceByName, &replicaSet),
+					resource.TestCheckResourceAttr(constant.DBaaSRedisDBReplicaSetResource+"."+constant.DBaaSReplicaSetTestDataSourceByName, replicaSetVersionAttribute, replicaSetVersionValue),
+					resource.TestCheckResourceAttr(constant.DBaaSRedisDBReplicaSetResource+"."+constant.DBaaSReplicaSetTestDataSourceByName, replicaSetReplicasAttribute, replicaSetReplicasValue),
+					resource.TestCheckResourceAttr(constant.DBaaSRedisDBReplicaSetResource+"."+constant.DBaaSReplicaSetTestDataSourceByName, replicaSetCoresAttribute, replicaSetCoresValue),
```

Comment:
@avirtopeanu-ionos `cores` is under `resources`, I don't think the attribute can be accessed directly, the path should look like `resources.0.cores`.

Reply:
thanks!